dojson/cli.py

```diff
@@ -73,6 +74,32 @@ def apply_rule(source, load, dump, rule, strict):
         ]))
 
 
+@cli.command("do_marcxml")
```

Comment:
> You can use `pass_context()` to get the context passed. This will only work if in addition to `ignore_unknown_options` you also set `allow_extra_args` as otherwise the command will abort with an error that there are leftover arguments. If you go with this solution, the extra arguments will be collected in `Context.args`.

Can you pass all extra args to `dump` function in `apply_rule` instead of creating new command?